docs/source/en/training/distributed_inference.md

```diff
@@ -52,6 +52,79 @@ To learn more, take a look at the [Distributed Inference with 🤗 Accelerate](h
 
 </Tip>
 
+### Device placement
+
+> [!WARNING]
+> This feature is experimental and its APIs might change in the future. 
+
+With Accelerate, you can use the `device_map` to determine how to distribute the models of a pipeline across multiple devices. This is useful in situations where you have more than one GPU.
+
+For example, if you have two 8GB GPUs, then using [`~DiffusionPipeline.enable_model_cpu_offload`] may not work so well because:
+
+* it only works on a single GPU
+* a single model might not fit on a single GPU ([`~DiffusionPipeline.enable_sequential_cpu_offload`] might work but it will be extremely slow and it is also limited to a single GPU)
```

Comment:
I think I made a comment yesterday about this but can't find it now. How does this PR help when a model does not fit in a single GPU?

Reply:
Yeah, that's a GitHub bug, apparently. I responded to it but it got disappeared as well. 

We offload to the CPU. In case there's not enough CPU memory, we error out. 

The next alternative would be to split the model across devices - GPU, CPU, and disk. This is a separate PR: https://github.com/huggingface/diffusers/pull/6396.